tox.ini

```diff
@@ -1,42 +1,62 @@
+# Generated from:
+# https://github.com/zopefoundation/meta/tree/master/config/pure-python
 [tox]
 envlist =
-   py27,
-   py35,
-   py36,
-   flake8,
-   coverage
+    lint,
+    py27,
+    py35,
+    py36,
+    py37,
+    py38,
+    py39,
+    coverage
 
 [testenv]
+usedevelop = true
+deps =
+    zope.testrunner
 commands =
-    {envbindir}/buildout -c {toxinidir}/buildout.cfg buildout:directory={envdir} buildout:develop={toxinidir} bootstrap
-    {envbindir}/buildout -c {toxinidir}/buildout.cfg buildout:directory={envdir} buildout:develop={toxinidir} install test
-    coverage run {envbindir}/test {posargs:-cv}
+    zope-testrunner --test-path=src []
+extras = test
+
+[testenv:lint]
+basepython = python3
 skip_install = true
 deps =
-    setuptools==33.1.1
-    zc.buildout
-    coverage
-setenv =
-    COVERAGE_FILE=.coverage.{envname}
+    flake8
+    check-manifest
+    check-python-versions
+commands =
+    flake8 src setup.py
+    check-manifest
+    check-python-versions
 
 [testenv:coverage]
-basepython = python3.6
-skip_install = true
-deps = coverage
-setenv =
-    COVERAGE_FILE=.coverage
+basepython = python3
+deps =
+    coverage
+    coverage-python-version
+    zope.testrunner
 commands =
-    coverage erase
-    coverage combine
+    coverage run -m zope.testrunner --test-path=src []
     coverage html
-    coverage xml
-    coverage report --fail-under=93
+    coverage report -m --fail-under=0
 
-[testenv:flake8]
-basepython = python3.6
-deps =
-    flake8
-    flake8-html
-    flake8-debugger
```

Comment:
I wonder if flake8-debugger is something we might want to add to the meta template.

(Probably not; if we have 100% test coverage, any stray `pdb.set_trace()` calls will get caught anyway.)

Reply:
As we have code reviews we will catch the other half. 😁